src/index.js

```diff
@@ -0,0 +1,41 @@
+// import _ from 'lodash';
```

Comment:
- [ ] I recommend to remove the commented code to look codebase more clean and also for best practices because you never have to send the commented code while pushing for production.